test/presenters/user_presenter_test.rb

```diff
@@ -23,10 +23,12 @@ def item
   end
 
   test "gravatar id" do
-    assert_equal '72dedd9e525e529e37b724e8aba4997f', pres.gravatar_id
+    pres.save!
```

Comment:
Does the test still work if you use `#save` instead of `#save!`? I always forget the difference between the two.